Classes/Queue/QueueManager.php

```diff
@@ -14,9 +14,11 @@
 use TYPO3\Flow\Annotations as Flow;
 use TYPO3\Flow\Object\ObjectManagerInterface;
 use Flowpack\JobQueue\Common\Exception as JobQueueException;
+use TYPO3\Flow\Utility\Arrays;
 
 /**
- * Queue manager
+ * A factory for queues
+ *
```

Comment:
Hmm, it's not just a factory, is it? It's also a registry. So, I guess it's still a "Manager". Whatever that means ;-)